JS-do-not-construct-regex-from-userinput/lomholdt_arp_content_script.js

```
@@ -0,0 +1,56 @@
+		if (c.indexOf(nameEQ) == 0) return decodeURIComponent(c.substring(nameEQ.length, c.length));
+	}
+	return null;
+}
+function writeCookie(name, value, hours) {
+	var hostname = window.location.hostname;
+	if (hours > 0) {
+		var date = new Date();
+		date.setTime(date.getTime() + (hours * 60 * 60 * 1000));
+		var expires = "; expires=" + date.toGMTString();
+	} else if(hours == -1) {
+		var expires = "; expires=" + new Date().toGMTString()+"; max-age=0";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'expires'. Consider to delete it or rename it.

'expires' is defined here:


(lint/suspicious/noRedeclare)